src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/AssessmentService.kt

```diff
@@ -22,4 +26,20 @@ class AssessmentService(
         application = cas2ApplicationEntity,
       ),
     )
+
+  fun updateAssessment(assessmentId: UUID, newAssessment: UpdateCas2Assessment): AuthorisableActionResult<ValidatableActionResult<Cas2AssessmentEntity>> {
```

Comment:
Is `assessmentId` a property of `UpdateCas2Assessment`?

Reply:
nope, the `assessmentId` is a path param e.g. `cas2/assessments/{assessmentId}`

Reply:
OK, trying to understand why `assessmentId` wouldn't be a property of the object we want to update.  Don't let me distract you though @Harriethw..